delphi/translators/for2py/types_ext.py

```diff
@@ -49,107 +47,133 @@ def __init__(self, val):
         self._val = float32(val)
 
     def __add__(self, other):
-        """ Addition """
+        """ Addition
```

Comment:
We don't need the ending triple quote on the next line...but this is not a priority if we are short on time.